geo/src/algorithm/mod.rs

```diff
@@ -183,7 +183,7 @@ pub use vincenty_length::VincentyLength;
 
 /// Calculate and work with the winding order of `Linestring`s.
 pub mod winding_order;
-pub use winding_order::WindingOrder;
+pub use winding_order::Winding;
```

Comment:
I got confused when I re-exported winding_order::WindingOrder from the algorithm module.

Breaking with the convention of our other modules, `winding_order::Winding` is the trait, while `winding_order::WindingOrder` is the type returned by that traits methods.